server/sockets.js

```diff
@@ -204,14 +212,6 @@ async function saveHistory(boardName, message) {
   board.processMessage(message);
 }
 
-function generateUID(prefix, suffix) {
-  var uid = Date.now().toString(36); //Create the uids in chronological order
-  uid += Math.round(Math.random() * 36).toString(36); //Add a random character at the end
-  if (prefix) uid = prefix + uid;
-  if (suffix) uid = uid + suffix;
-  return uid;
-}
-
```

Comment:
?

Reply:
Check my latest commit I have remove this function as this function is not used anywhere in the file and this function is created in the client-data/js/board.js file